api/pom.xml

```diff
@@ -161,7 +167,7 @@
                             <description>Jakarta Concurrency API documentation</description>
                             <doctitle>Jakarta Concurrency API documentation</doctitle>
                             <windowtitle>Jakarta Concurrency API documentation</windowtitle>
-                            <header><![CDATA[<br>Jakarta Concurrency API v${project.version}]]></header>
+                            <header><![CDATA[<br>Jakarta Concurrency API v${spec.specification.version}]]></header>
```

Comment:
Should this just be `spec.version`?

Reply:
It could be either.  The plugin before this does some verification between spec/project versions and spits out a set of properties that can be used elsewhere in the pom.xml.  So I used the verified value here.